lib/contentful.js

```diff
@@ -74,53 +79,57 @@ module.exports.getClient = function getClient() {
  */
 module.exports.contentfulError = function contentfulError(error) {
   const scope = error;
-  scope.message = `Contentful: ${error.message}`;
+  scope.message = `${ERROR_PREFIX} ${error.message}`;
 
   return scope;
 };
 
+module.exports.getFirstItemFromArray = function getFirstItemFromArray(array) {
```

Comment:
I was thinking we wouldn't need the function in helpers at all if we just called [`first`? ](http://underscorejs.org/#first)